src/mainview.cpp

```diff
@@ -94,6 +73,7 @@ void CMainView::OpenUrl( QString url )
 void CMainView::OpenProcess( QString execName, QStringList params )
 {
 	auto pProcess = new QProcess( this );
-	
-	pProcess->start( execName, params);
```

Comment:
Why did you change this?

Reply:
I was debugging a problem, and ended up changing it slightly, I can revert it back if you want.

Reply:
Please do, it's just less code honestly